src/ledger/parse/ledger.js

```diff
@@ -1,13 +1,15 @@
 /* @flow */
-'use strict';
+'use strict'; // eslint-disable-line
 const _ = require('lodash');
 const {removeUndefined, rippleTimeToISO8601} = require('./utils');
 const parseTransaction = require('./transaction');
 import type {GetLedger} from '../types.js';
 
 function parseTransactionWrapper(ledgerVersion, tx) {
-  const transaction = _.assign({}, _.omit(tx, 'metaData'),
-    {meta: tx.metaData});
+  const transaction = _.assign({}, _.omit(tx, 'metaData'), {
+    meta: tx.metaData,
+    ledger_index: ledgerVersion
```

Comment:
its another field that rippled includes in a 'tx' request, but not in a 'ledger' request - I need it at least for the ledger_index check in parseDeliveredAmount, I didn't check to see if it was expected and used elsewhere